src/modules/Header/components/Header.js

```diff
@@ -69,8 +70,18 @@ const HeaderBarLower = styled.div`
 `
 
 const HeaderUpperRight = styled.div`
-  padding: 14px 20px;
-  line-height: 18px;
+  display: flex;
+  flex-wrap: wrap;
+  padding-right: 20px;
+`
+
+const LanguageBar = styled.div`
+  padding: 14px 0;
+  @media ${MEDIA_QUERIES.MOBILE} {
```

Comment:
We can get rid of this whole media query section and pixel units by using Tailwind's predefined margin and padding classes and its responsive classes.